docs/data/migration/migration-charts-v7/migration-charts-v7.md

```diff
@@ -284,3 +284,43 @@ npx @mui/x-codemod@latest v8.0.0/charts/rename-sparkline-colors-to-color <path>
 ```
 
 For more complex cases, you may need to adjust the code manually. To aid you in finding these cases, the codemod adds a comment prefixed by `mui-x-codemod`, which you can search for in your codebase.
+
+## Replace `topAxis`, `rightAxis`, `bottomAxis`, and `leftAxis` props by `position` in the axis config
+
+The following props have been removed from the axis config: `topAxis`, `rightAxis`, `bottomAxis`, and `leftAxis`.
+
+Replaced the `topAxis`, `rightAxis`, `bottomAxis` and `leftAxis` props by a `position` prop in the axis config that accepts `'top' | 'right' | 'bottom' | 'left' | 'none'`.
```

Comment:
It looks a bit duplicated with the title. Maybe something more like

The following props have been removed `topAxis`, `rightAxis`, `bottomAxis`, and `leftAxis`.
To modify the axis placement, use the new `position` property in the axis config.
It accepts `'top' | 'right' | 'bottom' | 'left' | 'none'`.

Reply:
Maybe also at the end a small sentence to highlight why doing that:

> Notice this new API allows you to [stack multiple axes on the same side of the chart](link to the demo)